src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_item.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="sf-toolbar-block sf-toolbar-block-{{ name }} sf-toolbar-status-{{ status|default('normal') }} {{ additional_classes|default('') }}">
-    {% if link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
+    {% if link is defined and link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
```

Comment:
I don't know if this happens to others, but when I read this code: `if link is defined and link` it looks to me like something is missing (try to read it as a regular English phrase ... it ends abruptly and it's weird; it should be `if link is defined and link is true`).

But never mind about my comment ... let's keep your original code.